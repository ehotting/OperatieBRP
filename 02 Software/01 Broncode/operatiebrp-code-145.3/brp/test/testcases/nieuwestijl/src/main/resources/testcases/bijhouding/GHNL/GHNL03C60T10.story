Meta:
@auteur                 fuman
@status                 Klaar
@regels                 R2171
@usecase                UCS-BY.HG

Narrative:
Datum geboorte in Geboorte is verplicht

Scenario: Datum geboorte in Geboorte is verplicht andere velden in Geboorte gevuld
          LT: GHNL03C60T10



Given alle personen zijn verwijderd
Given enkel initiele vulling uit bestand /LO3PL-GHNL/GHNL-Mila.xls

When voer een GBA bijhouding uit GHNL03C60T10.xml namens partij 'Migratievoorziening'

Then is het foutief antwoordbericht gelijk aan /testcases/bijhouding/GHNL/expected/GHNL03C60T10.txt

Then is in de database de persoon met bsn 956803593 niet als PARTNER betrokken bij een HUWELIJK
Then is in de database de persoon met bsn 189975945 niet als PARTNER betrokken bij een HUWELIJK




