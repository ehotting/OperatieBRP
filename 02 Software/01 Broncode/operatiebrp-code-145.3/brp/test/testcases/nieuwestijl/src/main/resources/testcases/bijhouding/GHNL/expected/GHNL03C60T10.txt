cvc-complex-type.2.4.a: Invalid content was found starting with element 'brp:gemeenteCode'. One of '{"http://www.bzk.nl/brp/brp0200":datum}' is expected.